apps/web/src/pages/WorkforceAdjustment/WorkforceAdjustmentPage.tsx

```diff
@@ -0,0 +1,620 @@
+            }) + intl.formatMessage(commonMessages.dividingColon)}
+            <Link
+              external
+              href={
+                locale === "fr"
+                  ? "https://catalogue.csps-efpc.gc.ca/product?catalog=COR1-J15&cm_locale=fr"
+                  : "https://catalogue.csps-efpc.gc.ca/product?catalog=COR1-J15&cm_locale=en"
+              }
+            >
+              {intl.formatMessage({
+                defaultMessage: "Workforce Adjustment Process (COR1-J15)",
+                id: "XN7vop",
+                description: "Link text for workforce adjustment course",
+              })}
+            </Link>
+          </li>
+          <li>
+            <Link
+              external
+              href={
+                locale === "fr"
+                  ? "https://www.canada.ca/fr/secretariat-conseil-tresor/sujets/remuneration/conventions-collectives.html"
+                  : "https://www.canada.ca/en/treasury-board-secretariat/topics/pay/collective-agreements.html"
+              }
+            >
+              {intl.formatMessage({
+                defaultMessage: "Collective agreements for the public service",
+                id: "PqrUAO",
+                description:
+                  "Link text for public service collective agreements",
+              })}
+            </Link>
+          </li>
+        </Ul>
+        <Heading
+          color="primary"
+          icon={IdentificationIcon}
+          className="font-normal"
+        >
+          {intl.formatMessage({
+            defaultMessage: "CPA and non-CPA employees",
+            id: "edaphB",
+            description:
+              "Title for information about core public adminiatration employees related to worforce adjustment",
+          })}
+        </Heading>
+        <p className="my-6">
+          {intl.formatMessage({
+            defaultMessage:
+              "GC employees inside and outside the core public administration (CPA) can take advantage of job pools and new opportunities on the platform and can put themselves forward as a workforce affected employee in search of a new role (if they have received a WFA letter from their department). That said, alternations are only available between employees with indeterminate positions within the CPA.",
+            id: "GKE4/d",
+            description:
+              "Description of core public administration employees related to workforce adjustment",
+          })}
+        </p>
+        <Heading color="primary" icon={PuzzlePieceIcon} className="font-normal">
+          {intl.formatMessage({
+            defaultMessage: "How to use this service",
+            id: "9lhQKQ",
+            description:
+              "Title for how to share workforce adjustment information",
+          })}
+        </Heading>
+        <Instructions.List className="sm:grid-cols-3">
+          <Instructions.Step img={{ src: stepOne, darkSrc: stepOneDark }}>
+            <p className="font-bold">
```

Comment:
Instructions updated to add URLs

Figma

<img width="713" height="258" alt="image" src="https://github.com/user-attachments/assets/02e5f6db-656b-4c4b-9832-20c450b0a3fd" />

Reply:
Copy is still in flux so I have yet to add links.